cfpb/cfgov-refresh
# Remove old static templates

Doing Business with Us has moved to Wagtail, these old templates aren't
necessary any more.

## Removals

- The entire `doing-business-with-us/` directory
- The related styles
- Icon lists (they're no longer in CF).

## Testing

1. Run `gulp build` and ensure it doesn't error
2. Visit `http://localhost:8000/about-us/doing-business-with-us/` and it's children

## Todos

- ~~Pull out the flag code (:cough: @willbarton or @rosskarchner, want to push to this branch?)~~

## Checklist
